tests/components/ozw/test_websocket_api.py

```diff
@@ -119,6 +133,149 @@ async def test_websocket_api(hass, generic_data, hass_ws_client):
+        if option["Label"] != current_val
+    )
+    new_label = next(
+        option["Label"]
+        for option in config_param[ATTR_OPTIONS]
+        if option["Label"] != current_val and option["Value"] != new_val
+    )
+    await client.send_json(
+        {
+            ID: 14,
+            TYPE: "ozw/set_config_parameter",
+            NODE_ID: 39,
+            PARAMETER: config_param[ATTR_CONFIG_PARAMETER],
+            VALUE: new_val,
+        }
+    )
+    msg = await client.receive_json()
+    assert msg["success"]
+    await client.send_json(
+        {
+            ID: 15,
+            TYPE: "ozw/set_config_parameter",
+            NODE_ID: 39,
+            PARAMETER: config_param[ATTR_CONFIG_PARAMETER],
+            VALUE: new_label,
+        }
+    )
+    msg = await client.receive_json()
+    assert msg["success"]
+
+    # Test OZW Instance not found error
+    await client.send_json(
+        {ID: 16, TYPE: "ozw/get_config_parameters", OZW_INSTANCE: 999, NODE_ID: 1}
+    )
+    msg = await client.receive_json()
+    result = msg["error"]
+    assert result["code"] == ERR_NOT_FOUND
+
+    # Test OZW Node not found error
+    await client.send_json(
+        {
+            ID: 18,
+            TYPE: "ozw/set_config_parameter",
+            NODE_ID: 999,
+            PARAMETER: 0,
+            VALUE: "test",
+        }
+    )
+    msg = await client.receive_json()
+    result = msg["error"]
+    assert result["code"] == ERR_NOT_FOUND
+
+    # Test parameter not found
+    await client.send_json(
+        {
+            ID: 19,
+            TYPE: "ozw/set_config_parameter",
+            NODE_ID: 39,
+            PARAMETER: 45,
+            VALUE: "test",
+        }
+    )
+    msg = await client.receive_json()
+    result = msg["error"]
+    assert result["code"] == ERR_NOT_FOUND
+
+    # Test list value not found
+    await client.send_json(
+        {
+            ID: 20,
+            TYPE: "ozw/set_config_parameter",
+            NODE_ID: 39,
+            PARAMETER: config_param[ATTR_CONFIG_PARAMETER],
+            VALUE: "test",
+        }
+    )
+    msg = await client.receive_json()
+    result = msg["error"]
+    assert result["code"] == ERR_NOT_FOUND
+
+    # Test value type invalid
+    await client.send_json(
+        {
+            ID: 21,
+            TYPE: "ozw/set_config_parameter",
+            NODE_ID: 39,
+            PARAMETER: 3,
+            VALUE: 0,
+        }
+    )
+    msg = await client.receive_json()
+    result = msg["error"]
+    assert result["code"] == ERR_NOT_SUPPORTED
+
+    # Test invalid bitset format
+    await client.send_json(
+        {
+            ID: 22,
+            TYPE: "ozw/set_config_parameter",
+            NODE_ID: 39,
+            PARAMETER: 3,
+            VALUE: {ATTR_POSITION: 1, ATTR_VALUE: True, ATTR_LABEL: "test"},
+        }
+    )
+    msg = await client.receive_json()
+    result = msg["error"]
+    assert result["code"] == ERR_INVALID_FORMAT
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6285789)